manifests/kustomize/README.md

```diff
@@ -77,17 +60,62 @@ data:
             ...
 ```
 
-Or you can add it in one line with:
+Now you should be able to see the Model Registry UI in the Kubeflow Central Dashboard, and access to the Model Registry deployment in the profile namespace.
+
+### Uninstall
+
+To uninstall the Kubeflow Model Registry run:
 
 ```bash
-kubectl get configmap centraldashboard-config -n kubeflow -o json | jq '.data.links |= (fromjson | .menuLinks += [{"icon": "assignment", "link": "/model-registry/", "text": "Model Registry", "type": "item"}] | tojson)' | kubectl apply -f - -n kubeflow
-````
+# Uninstall Model Registry Instance
+PROFILE_NAME=<your-profile>
+for DIR in options/istio overlays/db ; do (cd $DIR; kustomize edit set namespace $PROFILE_NAME; kubectl delete -k .); done
+
+# Uninstall Model Registry UI
+kubectl delete -k options/ui/overlays/istio -n kubeflow
+```
+
+
+## Model Registry as a separate component Installation
+
```

Comment:
what is this and why is this needed? if were to add instructions to add additional MR instance ran as the profile again, IMO it is not acceptable to install without UI.

Reply:
This is the instructions **if you only wanna install Model Registry without UI**, we can rename it or completly remove it, it's basically what @pboyd did in https://github.com/kubeflow/website/pull/4042/files we can call it "standalone" here too, but it's basically that. If we should not install it wihtout UI I'm more than fine deleting this section.

Reply:
My understanding is that given that there's a full flegde API CLI users might wanna install MR without the UI in some cases.